app/Models/TnmsneInfo.php

```diff
@@ -19,3 +20,9 @@ class TnmsneInfo extends Model
         'neOpState',
     ];
 }
+   // ---- Define Relationships ----
+
+    public function alarms()
```

Comment:
A parse error occurred: Syntax error, unexpected T_PUBLIC on line 25 at column 4

Reply:
You have added it outside of the class